_posts/0000-01-05-Momoko-Sanchez.md

```diff
@@ -7,3 +7,5 @@ title: "Welcome to our slide deck!"
 {: .center}
 
 Caption here
+
+might want to look into that css later....
```

Comment:
remove the `Caption here` and replace it with your `might want to look` content 👍